datasets/standbedrijven/dataset.json

```diff
@@ -0,0 +1,581 @@
+          },
+          "ictSectorCode": {
+            "type": "string",
+            "title": "ictSectorCode",
+            "description": "Code die aangeeft in welke bedrijfssector van ICT de vestiging actief is"
+          },
+          "ictSectorOmschrijving": {
+            "type": "string",
+            "title": "ictSectorOmschrijving",
+            "description": "Omschrijving van de bedrijfssector van ICT waarin de vestiging actief is"
+          },
+          "toerismeCode": {
+            "type": "string",
+            "title": "toerismeCode",
+            "description": "Code die aangeeft in welke bedrijfssector van toerisme de vestiging actief is"
+          },
+          "toerismeOmschrijving": {
+            "type": "string",
+            "title": "toerismeOmschrijving",
+            "description": "Omschrijving van de bedrijfssector van toerisme waarin de vestiging actief is"
+          },
+          "horecaCode": {
+            "type": "string",
+            "title": "horecaCode",
+            "description": "Code die aangeeft in welke bedrijfssector van de horeca de vestiging actief is"
+          },
+          "horecaOmschrijving": {
+            "type": "string",
+            "title": "horecaOmschrijving",
+            "description": "Omschrijving van de bedrijfssector van de horeca waarin de vestiging actief is"
+          },
+          "nummeraanduidingIdentificatie": {
+            "type": "string",
+            "title": "nummeraanduidingIdentificatie",
+            "shortname": "nagId",
+            "description": "Landelijke identificerende sleutel van de nummeraanduiding"
+          },
+          "nummeraanduidingVolgnummer": {
+            "type": "integer",
+            "title": "nummeraanduidingVolgnummer",
+            "shortname": "nagVolgnummer",
+            "description": "Uniek volgnummer van de toestand van de nummeraanduiding"
+          },
+          "adresseerbaarobjectIdentificatie": {
+            "type": "string",
+            "title": "adresseerbaarobjectIdentificatie",
+            "shortname": "aotId",
+            "description": "Landelijke identificerende sleutel van adresseerbaar object (verblijfsobject, ligplaats, standplaats)"
+          },
+          "adresseerbaarobjectVolgnummer": {
+            "type": "integer",
+            "title": "adresseerbaarobjectVolgnummer",
+            "shortname": "aotVolgnummer",
+            "description": "Uniek volgnummer van de toestand van het adresseerbaar object"
+          },
+          "verblijfsobjectOppervlakte": {
+            "type": "integer",
+            "title": "verblijfsobjectOppervlakte",
+            "shortname": "votOppervlakte",
+            "description": "De gebruiksoppervlakte van het verblijfsobject waar de vestiging in is gevestigd conform hetgeen in NEN 2580 is vastgelegd omtrent gebruiksoppervlakte"
+          },
+          "geometrie": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
```

Comment:
In het geometrie veld wil je zo specifiek mogelijk zijn. Waarschijnlijk kan je hier "https://geojson.org/schema/MultiPolygon.json" of zelfs  "https://geojson.org/schema/Polygon.json" gebruiken.